lib/shelley/test/data/cardano-node-shelley/genesis.yaml

```diff
@@ -19,11 +19,11 @@ protocolParams:
   keyDeposit: 0
   keyDecayRate: 0
   nOpt: 100
-  rho: 0
+  rho: 0.178650067
```

Comment:
Documentation: https://github.com/input-output-hk/cardano-ledger-specs/blob/036ff8223310ecdaf7f0ed8c211c691cb0dcb8f3/shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/PParams.hs#L127,

(This particular value is copied from FF, but increased by some factors of 10)